docs/docusaurus/docs/core/run_validations/_choose_result_format/_BASIC.md

```diff
@@ -0,0 +1,10 @@
+When the `result_format` is set to `"BASIC"` the Validation Results of each Expectation includes a `result` dictionary with information providing a basic justification for why it failed or succeeded. The format is intended for quick feedback and it works well in Jupyter Notebooks.
+
+You can check the [Validation Results reference tables](#validation-results-reference-tables) to see what information is provided in the `result` dictionary.
+
+To create a `"BASIC"` result format configuration use the following code:
+
+```python
+basic_rf_dict = {}
```

Comment:
nit: it's less code and probably more clear to just oneline this: `basic_rf_dict = {"result_format": "BASIC"}`. We also have enums for these values. I'd probably use those unless you're trying to minimize imports.

Reply:
We are trying to minimize imports, and I was advised to use the dictionaries as the "happy path" over enums or passing in strings.  The reason behind the format of the code currently is that I initially intended to show the various other valid keys for each Result Format configuration -- but decided to forgo the additional code in favor of outlining the options in a table.  I'll update these examples; since they aren't currently in python scripts under test they'll likely be revised when that takes place regardless.